concent_api/core/views.py

```diff
@@ -854,7 +854,7 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
     )
     if cluster_storage_response.status_code == 200:
         return True
-    elif cluster_storage_response.status_code == 404:
+    elif cluster_storage_response.status_code == 404 or 401:
```

Comment:
It won't work this way. This will always return True. It will first check bool(cluster_storage_response.status_code == 404), and then do bool(401) which is True. Please do:
`elif cluster_storage_response.status_code in [401, 404]`